### DISCUSSION TITLE
Remotely Power on client when client is on Sleep mode

### DISCUSSION THREAD

**Discussion by LINOHWANG** [rustdesk/rustdesk#5583] [Q&A]
It would be good to have WOL or WOW feature : Wake on Lan or Wake on Wan using Magic packet using Mac address of NIC.
Anydesk have this feature to power on remote client when it is sleeping mode.

**Comment by SergeyMy** [comment]
hmm ... I remember exactly that there was such an item:
.
but disappeared somewhere
the Sciter version has it

PS
probably there were attempts to implement it, but abandoned

**Reply by Vyeche** [reply]
Could this be the related issue https://github.com/rustdesk/rustdesk/issues/981?

**Comment by deep-soft** [comment]
In Anydesk WOL works only if other computer, in the same LAN, is online (connected to Anydesk).

**Comment by PTrottier** [comment]
Not sure if it works in Sleep, but we used https://github.com/seriousm4x/UpSnap for WoL and WoW (via VPN) with great success.

Still, it would be nice if RustDesk had this built-in, but UpSnap is fantastic and meets our needs perfectly.
